### PR TITLE
niv: updating nixpkgs to pull latest packages

### PR DESCRIPTION
**Pull request by wiredhikari**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -13,6 +13,7 @@ in naersk.buildPackage {
       pkgs.pkg-config
       pkgs.perl
   ];
+gitSubmodules = true;
```
Comment (elsirion) [inline default.nix:16]: Why?

**Review by elsirion — COMMENTED** [review]
Good idea! I don't get the submodules thing, we aren't using any. Also, can you please reorder/squash commits so each commit passes CI?

File: default.nix (hunk 1 above)
Comment (wiredhikari) [reply default.nix:16]: We have recently been switching to git dependencies, so it will be good to add support by default in case we need in future

Comment (elsirion) [reply default.nix:16]: Ah, I see, that relates to the dependencies, agreed!

**Pull request merged**